pochoir/__main__.py

```diff
@@ -629,6 +627,69 @@ def plot_image(ctx, array, output, scale, units):
         title += f' [{units}]'
     pochoir.plots.image(arr, output, dom, title, scale=scale)
 
+@cli.command("plot-scatter3d")
+@click.option("-a", "--array", type=str, required=True,
+              help="Input array to plot")
+@click.option("-o", "--output",
+              type=click.Path(exists=False, dir_okay=False),
+              help="Output graphics file")
+@click.option("-g", "--gif",
+              default="no", type=click.Choice(["yes","no"]),
+              help="create gif image ")
+@click.pass_context
```

Comment:
This is not needed.  The extension to the `output` determines the file format.  If you pass `-o plot.gif` then you get a GIF.